pom.xml

```diff
@@ -107,6 +100,36 @@
                 </configuration>
             </plugin>
         </plugins>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-resources-plugin</artifactId>
+                    <version>3.1.0</version>
+                    <configuration>
+                        <escapeString>\</escapeString>
+                        <encoding>UTF-8</encoding>
+                    </configuration>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-archetype-plugin</artifactId>
+                    <version>3.1.3-SNAPSHOT</version>
```

Comment:
Having a `3.1.3-SNAPSHOT` version of the maven-archetype-plugin referenced here caused my local builds of the cleanup branch code to fail.  I'm assuming that it will create the same issue for most users too.

Changing version to `3.1.2` (latest full release) allowed the archetype project to build successfully (JDK 11).  That is the version of `archetype-packaging` being referred to in other changes for this PR (if that makes a difference).

I _think_ it should be set to `3.1.2` for this PR before it is merged ?

Reply:
Would you mind double checking this again? I fixed the SNAPSHOT repository definition to allow fetch a proper version of the plugin.

Reply:
The pom change to use `pluginRepositories` seemed to fix the issue for me (allowed proper download of the 3.1.3-SNAPSHOT).  :smile:

If that snapshot is preferred over 3.1.2 final (for whatever reason) the build does work now.

Reply:
SNAPSHOTs are fine but once we decide to prepare a release we must turn all SNAPSHOTs into proper versions. Hopefully they will release a new version till then :)